packages/react-core/src/components/Drawer/DrawerPanelContent.tsx

```diff
@@ -154,12 +164,14 @@ export const DrawerPanelContent: React.FunctionComponent<DrawerPanelContentProps
         } else {
           panel.current.style.setProperty('--pf-c-drawer__panel--FlexBasis', (newSize / max) * 100 + '%');
         }
+        currWidth = newSize;
       }
     }
   };
   const hidden = isStatic ? false : !isExpanded;
   return (
     <div
+      id={id}
```

Comment:
Is the id required for the callback to work?  Should we either generate an id when `onResize` is provided. or throw a console warning?

Reply:
ID isn't required for the callback to work, if it isn't provided the function will return undefined for that parameter. I added it only as a way for a user to keep track of resizing if there are multiple resizable drawers in a single page.